OneCommunityGlobal/HighestGoodNetworkApp
# Nahiyan finishing for Jay fix header UI mobile optimization

# Description
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/71467346/3e86512b-ed83-4a8c-9af6-60d25a170923)
Fix Header UI for smaller viewports
- Don't show header on login
- Space items on small viewports

## Related PRS (if any):
This is unrelated to backend PRs. Use latest `development` branch of HGNRest

## Main changes explained:
- Update `src/components/Header/Header.jsx` to return null if location pathname is `/login`
- Update `src/components/BMDashboard/BMHeader/BMHeader.jsx` to do the same because there is a comment in `route.js` saying to uncomment `BMHeader` component
- Update `src/components/Header/Header.css` to not use `column` for `flex-direction`

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. Log out and verify that header is not present
5. log as admin/volunteer (any) user
6. go to dashboard
7. resize viewport in developer tools so either width is below  760px
8. verify alignment of header is correct

## Screenshots or videos of changes:
<img width="1264" alt="no header on login" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/71467346/01b46810-2d1e-4fe5-8592-e61f8d880124">
Before
<img width="1277" alt="dev branch header" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/71467346/54adc503-d355-4a69-8d5b-dd8e34be7487">
After
<img width="1279" alt="new header spacing" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/71467346/f2e99bb8-a3cc-4ec2-9aa9-dce2b3744428">
